### PR TITLE
create alpha landing page

### PR DESCRIPTION
**Pull request by tatyana0219**

### CODE REVIEW

File: main/getting-started/alpha.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Agoric Alpha	
+Release Version: Adriatic
+
+![Alpha Cover](./assets/alpha-cover.png)
+
+We’re building components to power the DeFi ecosystem. Come take a peek under the hood and help us scale our next-gen infrastructure. The Alpha Adriatic release provides you access to our toolset, API, and public chain.
```
Comment (tyg) [inline main/getting-started/alpha.md:6]: I'm not sure is "scale" is the right word. "extend" or "grow" maybe?

I'd change "provides" to "gives"; shorter and stronger. 

I think "API" should be plural. We have several that are at least very separated in the docs (ERTP, Zoe, Wallet...).

File: main/getting-started/alpha.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+[Start Coding](https://agoric.com/documentation/getting-started/)
+[Join Community](https://discord.gg/gC9z6US)
```
Comment (tyg) [inline main/getting-started/alpha.md:9]: "Join The Community" or "Join Our Community" or "Join The Agoric Community"

File: main/getting-started/alpha.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+---------------------
+
+## What you can build in Alpha Adriatic 
+
+Numerous transaction components -- what we think of as DeFi Legos -- are available in the Alpha Adriatic release. We’ll be adding to the pool with our growing community’s help and participation in the coming months as we scale contributions. The initial slate of elements includes the following:
```
Comment (tyg) [inline main/getting-started/alpha.md:15]: I know I saw a discussion of the trademark issues fly by, but I think capitalizing "Legos" is asking for it. As it's text and not a title, capitalization means it's a proper noun; i.e. the Danish toy brick. If we're going to claim generic use, it should be "legos"; i.e. you don't say you'll make a Xerox, but that you'll make a xerox if you want a generic use claim. 

I'd redo the last two sentences as "In the coming months, with your and the rest of our community's help, we'll be adding more to this initial list:

File: main/getting-started/alpha.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+Numerous transaction components -- what we think of as DeFi Legos -- are available in the Alpha Adriatic release. We’ll be adding to the pool with our growing community’s help and participation in the coming months as we scale contributions. The initial slate of elements includes the following: 
+
+![Automated Market Maker](./assets/amm.png)
+### Automated Market Maker
+Build a smart contract that acts as an on-chain automated DEX with liquidity. 
+
+![Autoswap](./assets/autoswap.png)
+### Autoswap
+Tap into our Uniswap-like, constant-product AMM that executes upon finding a match in the current liquidity pool.
```
Comment (tyg) [inline main/getting-started/alpha.md:23]: Unless you're sure all of the audience will know what it is/stands for (I don't), give an expansion for AMM i.e. "AMM (Automatic Metal Model)"

File: main/getting-started/alpha.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+![Covered Call](./assets/covered-call.png)
+### Covered Call
+Create an option contract that enables the holder to buy a specific, previously escrowed asset at a predetermined strike price (before expiration).
```
Comment (tyg) [inline main/getting-started/alpha.md:27]: "specific, previously escrowed, asset" (add , after escrowed)

File: main/getting-started/alpha.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+![Simple Exchange](./assets/simple-exchange.png)
+### Simple Exchange
+A basic exchange with an order book for one asset, priced in a second asset.
+
```
Comment (tyg) [inline main/getting-started/alpha.md:32]: Do we want to add a final ### Additional Sample Components
"We also provide examples of:  (list of just names)". I know we're got more than just these.

File: main/getting-started/alpha.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+## Anatomy of a Smart Contract
+A call option is the right (with no obligation) to buy digital assets at a predetermined price, called the strike price. This call option is "covered," meaning that the owner of the digital assets must put the assets in escrow. Escrow guarantees that the assets can be transferred without relying on the owner of the digital assets to keep their promise. The call option has an expiration date, when the opportunity is cancelled. The owner of the digital assets cannot remove the assets from escrow before the expiration date. Any number of assets of different brands can be escrowed under different keywords. The proposal must have an exit record with the key "afterDeadline":
```
Comment (tyg) [inline main/getting-started/alpha.md:34]: "owner of the digital assets" -> "digital assets owner"
"relying on the owner of the digital assets" -> "relying on the owner"
"The owner of the digital assets cannot remove the assets" -> "The digital assets owner cannot remove their assets"

"Any number of assets of different brands can be escrowed under different keywords. The proposal must have an exit record with the key "afterDeadline": "    These are using various names and concepts that we've defined and newbies won't know. Rewrite to be conceptual for understanding; "You can escrow different kinds of assets under different keys. The option must specify an expiration time." [although that last is a bit redundant with "The call option has an expirations date"]

File: main/getting-started/alpha.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+
+```
+{
+    give: { StrikePrice: moola(25) },
+    want: { Asset: concertTicket("E4") }, 
+    exit: { afterDeadline: { deadline: time, timer: myTimer } },
+}
+```
+
+This deadline serves as the expiration date for the covered call option. After this deadline, if the option has not been exercised, the underlying assets are automatically paid out to the creator of the contract as a refund.
```
Comment (tyg) [inline main/getting-started/alpha.md:44]: Rewrite to: "If the option has not been exercised by the deadline, as specified in the above code example,  its underlying assets are paid out to the contract's creator as a refund of their escrowed assets." (I really want to include a {"...") few words definition of "exercised" after it, but I'm not coming up with anything brief enough).

File: main/getting-started/alpha.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+    exit: { afterDeadline: { deadline: time, timer: myTimer } },
+}
+```
+
+This deadline serves as the expiration date for the covered call option. After this deadline, if the option has not been exercised, the underlying assets are automatically paid out to the creator of the contract as a refund. 
+
+[Start Building](https://agoric.com/documentation/getting-started/)
+
+## Focused on the developer experience
+Agoric is a company of engineers, and we've tailored the developer experience based on our decades of activity in open-source software. Here’s what you can expect in Alpha.
```
Comment (tyg) [inline main/getting-started/alpha.md:49]: "Agoric is a company of engineers. We've based our developer experience on our decades of open-source software experience. In our Alpha, you can expect:

File: main/getting-started/alpha.md [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+## Focused on the developer experience
+Agoric is a company of engineers, and we've tailored the developer experience based on our decades of activity in open-source software. Here’s what you can expect in Alpha. 
+
+* VS Code with debugging support
+* Ability to build and test locally
+* Types included to accelerate your local development
+* Rapid testing iteration with our AVA infrastructure
+
+### Stuck on something?
+Chat with our engineers on [Agorc Discord](https://discord.gg/gC9z6US).
```
Comment (tyg) [inline main/getting-started/alpha.md:57]: "Agorc" -> "Agoric"

File: main/getting-started/alpha.md [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+
+Want to connect on socials?
```
Comment (tyg) [inline main/getting-started/alpha.md:59]: If we're only listing one, I'd go with "social media" rather than "socials"

**Review by dtribble — APPROVED** [review]
Ran the build. Moved the alpha nav to the top. Everything works and navigates.

**Pull request merged**